js/packages/botSocket/botSocket.js

```diff
@@ -99,6 +96,7 @@ class botSocket extends EventEmitter {
         });
     }
 
+    // TODO: document us PLZZZZZZ
```

Comment:
Consider just adding a comment to document it now

Reply:
Done